timemachine/cpp/src/local_md_potentials.cu

```diff
@@ -170,9 +189,9 @@ void LocalMDPotentials::_setup_free_idxs_given_reference_idx(
     gpuErrchk(cudaStreamSynchronize(stream));
 
     // The row indices is all of the free indices, which excludes the reference
-    const int num_row_idxs = p_num_selected_.data[0];
+    int num_row_idxs = p_num_selected_.data[0];
     // The col indices is all indices, except for the free, including the reference which is frozen.
-    const int num_col_idxs = N_ - num_row_idxs;
+    int num_col_idxs = N_ - num_row_idxs;
```

Comment:
nit: were these changes intentional? (don't immediately see why these can't be `const`)

Reply:
Was done in a previous implementation, put back in [2a2a934](https://github.com/proteneer/timemachine/pull/1014/commits/2a2a9344ad5eee81fd1d7ae5bf95efeb003d4121)